packages/web/utils/ensHelpers.ts

```diff
@@ -0,0 +1,17 @@
+import { ethers } from 'ethers';
+
+const mainnetProvider = new ethers.providers.JsonRpcProvider(
+  'https://eth.llamarpc.com',
+);
+
+export const getAddressFromName = async (ens: string) => {
+  const address = await mainnetProvider.resolveName(ens);
+  const data = address?.length === 42 ? address : ens; 
+  return data;
```

Comment:
```suggestion
  return address?.length === 42 ? address : ens; 
```

Reply:
Followed through with this, in latest commit